vendor/assets/javascripts/jasmine-jquery.js

```diff
@@ -0,0 +1,832 @@
+/*!
```

Comment:
This file has been included by accident? It should be pulled into via the asset pipeline from the `jasmine-jquery-rails` gem